plugins/playstation/src/playstation-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.PlayStation : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "playstation";
+	private const string MIME_TYPE = "application/x-cue";
```

Comment:
I checked the `.bin` files of games and here is what I found.

## Games with only one .bin track

All these games have a keyfile at offset 0xD368, the lines all end by 0x0D 0x0A.

The order of the keys seem to always be the same, though on a game the TCB one didn't appear. Spaces are optional. All seem to start by `BOOT = cdrom:` with or without the spaces.

It seems possible to extract the ID of the game from the BOOT line.

Ape Escape at 0xD368
```
BOOT=cdrom:\SCUS_944.23;1
TCB=20
EVENT=20
STACK=801FFFF
```

Final Fantasy IV at 0xD368
```
BOOT = cdrom:\SCES_038.40;1
TCB = 4
EVENT = 16
STACK = 801fff0
```

Final Fantasy V at 0xD368
```
BOOT = cdrom:\SCES_138.40;1
TCB = 4
EVENT = 16
STACK = 801fff0
```

## Games with multiple .bin tracks

On the multi `.bin` games I checked, only the first `.bin` track had the keyfile and it was located at 0xD3A8 of 0xD368.

Sympgony of the Night at 0xD3A8
```
BOOT = cdrom:SLUS_000.67;1
TCB = 4
EVENT = 16
STACK = 801fff0
```

Tekken 3 at 0xD3A8
```
BOOT = cdrom:\TEKKEN3\SLUS_004.02;1
EVENT = 10
STACK = 801FFFF
```